src/frontend/Komponenter/Personoversikt/Vedtaksperioderoversikt.tsx

```diff
@@ -116,20 +134,69 @@ const VedtaksperioderTabell: React.FC<{ andeler: AndelHistorikk[] }> = ({ andele
     );
 };
 
-const Vedtaksperioder: React.FC<{ fagsakId: string }> = ({ fagsakId }) => {
+const Vedtaksperioder: React.FC<{ fagsakId: string; tomBehandlingId?: string }> = ({
+    fagsakId,
+    tomBehandlingId,
+}) => {
     const periodeHistorikkConfig: AxiosRequestConfig = useMemo(
         () => ({
             method: 'GET',
             url: `/familie-ef-sak/api/perioder/fagsak/${fagsakId}/historikk`,
+            params: tomBehandlingId && { tomBehandlingId },
         }),
-        [fagsakId]
+        [fagsakId, tomBehandlingId]
     );
     const perioder = useDataHenter<AndelHistorikk[], null>(periodeHistorikkConfig);
-
     return (
         <DataViewer response={{ perioder }}>
-            {({ perioder }) => <VedtaksperioderTabell andeler={perioder} />}
+            {({ perioder }) => {
+                return <VedtaksperioderTabell andeler={perioder} />;
+            }}
+        </DataViewer>
+    );
+};
+
+const Vedtaksperioderoversikt: React.FC<{ fagsakId: string }> = ({ fagsakId }) => {
+    const [tomBehandlingId, settTomBehandlingId] = useState<string>();
```

Comment:
Usikker på om jeg skjønner hva dette betyr - `tomBehandlingId` != `emptyBehandlingId` ? Er det `tilOgMedBehandlingId` vi mener her 😂 

Kan vi isåfall komme opp med et mer lettfattelig navn? 
`perioderTilOgMedBehandling` eller noe i den duren? 🤔

Reply:
Ja, kan godt bruke `const [valgtBehandlingId, settValgtBehandlingId] = useState<string>();` her og bruke
`const Vedtaksperioder: React.FC<{ fagsakId: string; perioderTilOgMedBehandling?: string }> = ({`

API'et burde kanskje også endres til `tilOgMedBehandlingId`?

Reply:
Tror kanskje det, mitt hode tenkte iallefall umiddelbart på "empty" = "tom" :)